saltswap/wrappers.py

```diff
@@ -378,6 +382,164 @@ def update(self, nattempts=None, chemical_potential=None, saltmax=None):
         self.swapper.update(self.context, nattempts=nattempts, cost=chemical_potential, saltmax=saltmax)
 
 
+class SAMSSalinator(Salinator):
+    """
+    Class to perform self-adjusted mixture sampling over salt-pair numbers between a minimum and maximum number.
+
+    Example
+    -------
+    Running SAMS on a box of TIP3P water. This sort of simulations that be used to calibrate the chemical potential for
+    the TIP3P water model and a particular set of non-bonded parameters.
+
+    >>> from simtk import openmm, unit
+    >>> from openmmtools import testsystems
+    >>> from openmmtools import integrators
+    >>> from saltswap.record import Record
+
+    Set the thermondynamic parameters
+    >>> temperature = 300.0 * unit.kelvin
+    >>> pressure = 1.0 * unit.atmospheres
+    >>> salt_concentration = 0.2 * unit.molar
+
+    Extract the test system:
+    >>> testobj = getattr(testsystems, 'WaterBox')
```

Comment:
The ion parameters are hard-coded in `saltswap.Swapper`. I'll make this clearer in the `Salinator` class.